py/mpconfig.h

```diff
@@ -1042,6 +1042,11 @@ typedef double mp_float_t;
 #define MICROPY_PY_BUILTINS_MEMORYVIEW_ITEMSIZE (0)
 #endif
 
+// Add bytes.hex and bytes.fromhex
+#ifndef MICROPY_PY_BUILTINS_STR_HEX
+#define MICROPY_PY_BUILTINS_STR_HEX (MICROPY_CONFIG_ROM_LEVEL_AT_LEAST_EXTRA_FEATURES)
+#endif
```

Comment:
move this up to just after `MICROPY_PY_BUILTINS_STR_COUNT` to keep the str config options grouped together

Reply:
I've renamed it to MICROPY_PY_BUILTINS_BYTES_HEX because it no longer applies to str.